dll/win32/shimgvw/shimgvw.c

```diff
@@ -1109,6 +1095,12 @@ ImageView_CreateWindow(HWND hwnd, LPWSTR szFileName)
     WCHAR szInitialFile[MAX_PATH];
     HWND hMainWnd;
     MSG msg;
+    HACCEL hKbdAccel;
+    INITCOMMONCONTROLSEX Icc;
+
+    Icc.dwSize = sizeof(INITCOMMONCONTROLSEX);
+    Icc.dwICC = ICC_WIN95_CLASSES;
```

Comment:
```suggestion
    INITCOMMONCONTROLSEX Icc = { sizeof(Icc), ICC_WIN95_CLASSES };
```

Reply:
I don't agree. I prefer to see what I'm initializing.

Reply:
You can do
`{ .dwSize = sizeof(Icc), .dwICC = ICC_WIN95_CLASSES }`
:)